harness/determined/searcher/_search_runner.py

```diff
@@ -80,18 +80,28 @@ def _get_operations(self, event: bindings.v1SearcherEvent) -> List[searcher.Oper
             # duplicate completion accounting already performed by master
             logger.info(
                 f"validationCompleted({event.validationCompleted.requestId},"
-                f" {event.validationCompleted.metric})"
+                f" {event.validationCompleted.allMetrics})"
```

Comment:
is it wrong to print just one of `allMetrics` vs `metric`?  How do you know which one to print?

Reply:
You mean I can be receiving `metric` xor `allMetrics`? I think you are right.